tools/scoring/score.py

```diff
@@ -562,34 +646,136 @@ def project(normal,other):
             [normal,other],
             keys=['normal','other']).drop_duplicates(
                     keep=False,
-                    subset=['year','division','solver','Rank'])
+                    subset=['solver','rank'])
     if different.empty:
         return different
     other_different = different.loc['other']
     return other_different
 
+# Do not consider solver variants for determining if a division is
+# competitive.
+# Uses solver_str(solver) to get the base version of solver.
+def is_competitive_division(solvers):
```

Comment:
We should probably use https://github.com/SMT-COMP/smt-comp/blob/master/2019/registration/solvers_divisions_final.csv to identify solver variants. But we can add that after this is merged in.